### PR TITLE
Feat: creators hero

### PR DESCRIPTION
**Pull request by krishguptadev**
- [x] Adds a hero
- [x] Improve dev-workflow using yarn!

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/88"></a>

### CODE REVIEW

File: src/components/Creators/Hero.astro [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  </div>
+  <div>
+    <h3 class="font-black font-display text-6xl mt-4">
+      Create public goods to Earn.
+    </h3>
+    <p class="max-w-prose my-6">
+      Creators can start monetization of any kind of works by issuing their own
+      social token (ERC-20) and then promoting them to the community. It's like
+      a "stock company" with all the features you need to grow your business,
+      including efficient management of ownership and distribution of revenue.
```
Comment (Panquesito7) [inline src/components/Creators/Hero.astro:31]:
```suggestion
      Creators can start monetization of any kind of works by issuing their 
      social tokens (ERC-20) and then promoting them to the community. It's like
      a "stock company" with all the features you need to grow your business,
      including efficient management of ownership and distribution of revenue.
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please update the `.gitpod.yml` file to use Yarn commands.

**Comment by Panquesito7** [comment]
The `.gitpod.yml` file should be updated to use Yarn now. Let's make that after this PR is merged. 🙂

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🚀

**Review by shuklaritvik06 — APPROVED** [review]

**Review by vinzvinci — COMMENTED** [review]
LGTM 👍

**Pull request merged**